compiler/z/codegen/BinaryCommutativeAnalyser.hpp

```diff
@@ -69,20 +69,11 @@ class TR_S390BinaryCommutativeAnalyser : public TR_Analyser
                         TR::InstOpCode::S390BranchCondition fBranchOpCond = TR::InstOpCode::COND_NOP,
                         TR::InstOpCode::S390BranchCondition rBranchOpCond = TR::InstOpCode::COND_NOP);
 
-   void genericLongAnalyser(TR::Node       *root,
-                            TR::InstOpCode::Mnemonic lowRegToRegOpCode,
-                            TR::InstOpCode::Mnemonic highRegToRegOpCode,
-                            TR::InstOpCode::Mnemonic lowMemToRegOpCode,
-                            TR::InstOpCode::Mnemonic highMemToRegOpCode,
-                            TR::InstOpCode::Mnemonic copyOpCode);
-
    void integerAddAnalyser(TR::Node       *root,
                            TR::InstOpCode::Mnemonic regToRegOpCode,
                            TR::InstOpCode::Mnemonic memToRegOpCode,
                            TR::InstOpCode::Mnemonic copyOpCode);
 
-   void longAddAnalyser(TR::Node *root, TR::InstOpCode::Mnemonic copyOpCode);
```

Comment:
These analyzers become dead as they're only used for when 64-bit values must go into register pairs, which is no longer true on any target.